apps/tx-builder/src/components/forms/fields/TextareaContractField.tsx

```diff
@@ -0,0 +1,10 @@
+import { TextFieldInputProps } from '@gnosis.pm/safe-react-components/dist/inputs/TextFieldInput';
+import TextContractField from './TextContractField';
+
+const DEFAULT_ROWS = 4;
+
+const TextareaContractField = (props: TextFieldInputProps) => {
+  return <TextContractField {...props} multiline rows={DEFAULT_ROWS} />;
```

Comment:
should rows be editable?

Reply:
Good point! 

Now `rows` are editable:

```
<TextContractField {...props} multiline rows={props.rows || DEFAULT_ROWS} />;
```